tasks/configure.js

```diff
@@ -1,33 +1,37 @@
 'use strict';
 
-let packageJson = require(process.cwd() + '/package.json');
-let findService = require('../lib/find-service');
-let herokuAuthToken = require('../lib/heroku-auth-token');
-let normalizeName = require('../lib/normalize-name');
-let vault = require('../lib/vault');
-let fetchres = require('fetchres');
+const packageJson = require(process.cwd() + '/package.json');
+const findService = require('../lib/find-service');
+const herokuAuthToken = require('../lib/heroku-auth-token');
+const normalizeName = require('../lib/normalize-name');
+const vault = require('../lib/vault');
+const pipelines = require('../lib/pipelines');
+const HerokuConfigVars = require('../lib/heroku-config-vars');
 
 const FORBIDDEN_ATTACHMENT_VARIABLES = [
 	'DATABASE_URL'
 ];
 
 const DEFAULT_REGISTRY_URI = 'https://next-registry.ft.com/v2/';
 
-const getServiceData = (source, registry) => fetch(registry)
-	.then(response => response.json())
-	.then(json => {
-		const serviceData = findService(json, normalizeName(source));
-		if (!serviceData) {
-			throw new Error('Could not find a service in the registry, with `name` or `systemCode`, matching ' + source + '. Please check the service registry.');
-			return false;
-		}
-		else {
-			return serviceData;
-		}
-	});
+const getServiceData = (source, registry) => {
+
+	return fetch(registry)
+		.then(response => response.json())
+		.then(json => {
+			const serviceData = findService(json, normalizeName(source));
+			if (!serviceData) {
+				throw new Error('Could not find a service in the registry, with `name` or `systemCode`, matching ' + source + '. Please check the service registry.');
+				return false;
+			}
+			else {
+				return serviceData;
+			}
+		});
+};
 
-function fetchFromVault (source, target, serviceData) {
-	const path = serviceData.config.replace('https://vault.in.ft.com/v1/','');
+function fetchFromVault (serviceData) {
```

Comment:
Should we convert this to an arrow function as well, for consistency with the rest of the file?

Maybe even async?

Reply:
Yep, I will change it! We didn't change it to async yet as felt it would be worth writing tests first...don't know which is scarier though